src/views/UserProfileView.vue

```diff
@@ -154,9 +154,11 @@
         userCommentCount: 0,
         profile: {
           biography: '',
-          displayName: ''
+          displayName: '',
+          image: false,
         },
         avatar,
+        BACKEND_URL : "https://raendom-backend.z52da5wt.xyz",
```

Comment:
why is the backend needed here?

Reply:
because the image is served from there.

Reply:
it should be fetched via backendInstance

Reply:
The default avatar will also be served via the backend, so I will refactor this.